modular_skyrat/modules/modular_items/lewd_items/code/lewd_organs/_genital.dm

```diff
@@ -0,0 +1,4 @@
+/obj/item/organ/external/genital
+	var/datum/reagents/internal_fluids
+	var/list/contained_item
+	var/obj/item/inserted_item //Used for toys
```

Comment:
These should all be documented, and the list should be called `contained_items` rather than `contained_item`, in order to make sense.

Reply:
That list is unused, which makes it even worse.